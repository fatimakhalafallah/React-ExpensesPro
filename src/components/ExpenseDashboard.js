import React from 'react';
import  ExpenseList  from './ExpenseList';
import ExpenseFilter from './ExpenseListFilters';

const ExpenseDashboardPAge =()=>(
    <div>
        <p> This is the dashboard page</p>
        <ExpenseFilter/>
        <ExpenseList/>
    </div>
);


export default ExpenseDashboardPAge;
