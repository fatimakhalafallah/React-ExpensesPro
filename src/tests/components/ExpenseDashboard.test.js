import React from 'react';
import ExpenseDashboard from '../../components/ExpenseDashboard';
import { shallow } from 'enzyme';

test('should render dashboard',()=>{

    const wrapper = shallow(< ExpenseDashboard/>)
    expect(wrapper).toMatchSnapshot();
})