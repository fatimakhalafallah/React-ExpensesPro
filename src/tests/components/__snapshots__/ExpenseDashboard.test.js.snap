// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard 1`] = `
<div>
  <p>
     This is the dashboard page
  </p>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
